JackieSL1/mini-survey-monkey
# Generic Sharing Link Displayed on Collect Responses page

# Description

Displays a generic link ("r/1") when user gets to collect responses page. This link then takes the user to a blank page, but will eventually take them to where they can fill out the survey

## Issues

#24 (doesn't close yet)

# How to test


1. http://localhost:8080/login
2. Create a new survey
3. Click Collect Responses button
4. Confirm you are directed to collect page
5. Copy the link displayed
6. Paste the link into a new tab
7. Confirm that you are directed to a blank page

Integration tests:
* Run `mvn test`

# Checklist
**I've added tests**


**I've updated the DB schema and UML diagram**
